docs/local_setup_sans_docker.md

```diff
@@ -17,6 +29,25 @@ brew update && brew upgrade ruby-build
 rbenv install
 ```
 
+Next, we'll want to make sure our ruby environment is properly initialized. Open your `~/.zshrc` (or `~/.bash` if that's your jam):
+
+```
+open ~/.zshrc
```

Comment:
📝 you could instead do `echo '\n eval "$(rbenv init -)"' >> ~/.zshrc \n` which would add the line to the end of their `.zshrc` file